icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2254,23 +2255,42 @@ class U_I18N_API NumberFormatterSettings {
     Derived usage(StringPiece usage) &&;
 
 #ifndef U_HIDE_DRAFT_API
-#ifndef U_HIDE_INTERNAL_API
+    /**
+     * Specifies the DisplayOptions. Such as `GrammaticalCase` specifies the desired case for
+     * a unit formatter's output (e.g. accusative, dative, genitive).
+     *
+     * @param displayOptions
+     * @return The fluent chain.
+     * @draft ICU 72
+     */
+    Derived displayOptions(const DisplayOptions &displayOptions) const &;
+
+    /**
+     * Overload of displayOptions() for use on an rvalue reference.
+     *
+     * @param displayOptions
+     * @return The fluent chain.
+     * @draft ICU 72
+     */
+    Derived displayOptions(const DisplayOptions &displayOptions) &&;
+#endif // U_HIDE_DRAFT_API
+
+#ifndef U_HIDE_DEPRECATED_API
     /**
      * Specifies the desired case for a unit formatter's output (e.g.
      * accusative, dative, genitive).
      *
-     * @internal ICU 69 technology preview
+     * @deprecated ICU 72 technology preview
```

Comment:
not deprecated. internal if it's still a technology preview. otherwise draft.

Reply:
`@deprecated` to prevent the user from using it and they should the new `displayOptions` method. after a cycle. we will remove it to help in smooth migrations.